public-api/vx/node2.php

```diff
@@ -218,17 +218,56 @@ function nodeAPI_Filter( $nodes, &$more_nodes ) {
 	// if not logged in, where will be blank
 	$RESPONSE['where'] = nodeComplete_GetWhereIdCanCreate(userAuth_GetID());
 }],
-["node2/what", API_GET | API_AUTH | API_CHARGE, function(&$RESPONSE, $HEAD_REQUEST) {
-	$parent_id = intval(json_ArgShift());
-	if ( !$parent_id )
+// Figure out what's featured, and what you made for it
+["node2/what", API_GET | API_CHARGE, function(&$RESPONSE, $HEAD_REQUEST) {
```

Comment:
why is there no longer auth required to figure out 'what you made'?

Reply:
alright I understand why it works but with that comment on the line above it looks a bit scary..

Reply:
Fair enough, the description can be improved.